dwave/cloud/client.py

```diff
@@ -221,10 +224,17 @@ def from_config(cls, config_file=None, profile=None, client=None,
             token (str, default=None):
                 API authorization token.
 
-            solver (str, default=None):
-                Default :term:`solver` to use in :meth:`~dwave.cloud.client.Client.get_solver`.
-                If undefined, :meth:`~dwave.cloud.client.Client.get_solver` will return the
-                first solver available.
+            solver (dict/str, default=None):
+                Default :term:`solver` features to use in :meth:`~dwave.cloud.client.Client.get_solver`.
+
+                Defined via dictionary of solver feature constraints
+                (see :meth:`~dwave.cloud.client.Client.get_solvers`). For backward
+                compatibility, string solver name is also accepted, and it's
+                transformed to ``{"name": <solver name>}``.
+
+                If undefined, :meth:`~dwave.cloud.client.Client.get_solver` will use
+                solver definition from environment variables, configuration file, or
+                fallback to the first available online solver.
```

Comment:
If undefined, :meth:`~dwave.cloud.client.Client.get_solver` uses a solver definition from environment variables, a configuration file, or falls back to the first available online solver.